Form-Controls/styles.css

```diff
@@ -0,0 +1,74 @@
+
+body{
+    background: #F68E5F;  
+    text-align: center;
+    width: 30%;
+    margin:auto;
+}
+main{
+    display: flex;
+    flex-direction: column;
+    align-items:center  ;
+    justify-content: center;
+    background: #F5DD90;
+    border-radius: 20px;
+    padding-bottom: 20px;
+}
+
+#heading{
+    margin: 0 0 50px 0;
+}
+
+input{
+    width: 100%;
+    padding: 12px 20px;
+    margin: 8px 0;
+    box-sizing: border-box;
+    background-color: white;
+    color: black;
+  }
+
+
+#name, #mail, #date{
+    height:50px;
+    font-size:14pt;
+    border-radius: 20px;
+}
+
+#color{
+    display:flex;
+    flex-direction: row;
+}
+
+#size{
+    width: 320px;
+    height:50px;
+    border-radius: 20px;
+    text-align: center;
+    font-size: large;
+}
+
+.button{
+    margin: 20px auto;
+    align-items: center;
+    background-image: linear-gradient(135deg, #f34079 40%, #fc894d);
+    border: 0;
+    border-radius: 10px;
+    box-sizing: border-box;
+    color: #fff;
+    display: flex;
+    flex-direction: column;
+    font-size: 16px;
+    font-weight: 700;
+    height: 54px;
```

Comment:
```suggestion
```

Instead of setting the height of a button, fill it with text and use padding or gap to achieve your look.